src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
+ *
+ * @description
+ *
+ * Use this directive to force the angular.element library.  This should be used to force
+ * either jqLite by leaving ng-jq blank or setting the name of the jquery variable under window
+ * (eg. jQuery).
+ *
+ * This directive is global for the whole of the angular library, hence the directive can only
+ * be added to the top-most HTML element.
```

Comment:
@mzgol @petebacondarwin I just tried merging the upstream master to my local without issue, however, the build is still failing even after an npm/bower install/update.  Same error as the CI build (unless there's something different on the build server that I'm missing locally):

```
error:   TypeError: Cannot read property '0' of undefined
    at _.groupBy.tap.tap.docTypes.directive (/home/michelb/Work/angular.js/docs/config/processors/pages-data.js:65:43)
    at Function.tap (/home/michelb/Work/angular.js/node_modules/lodash/dist/lodash.js:6448:7)
    at lodash.ctor.(anonymous function) [as tap] (/home/michelb/Work/angular.js/node_modules/lodash/dist/lodash.js:5924:31)
    at /home/michelb/Work/angular.js/docs/config/processors/pages-data.js:62:14
    at /home/michelb/Work/angular.js/node_modules/lodash/dist/lodash.js:3513:29
    at forOwn (/home/michelb/Work/angular.js/node_modules/lodash/dist/lodash.js:2105:15)
    at Function.map (/home/michelb/Work/angular.js/node_modules/lodash/dist/lodash.js:3512:9)
    at lodash.ctor.(anonymous function) [as map] (/home/michelb/Work/angular.js/node_modules/lodash/dist/lodash.js:5924:31)
    at navGroupMappers.api (/home/michelb/Work/angular.js/docs/config/processors/pages-data.js:53:10)
    at /home/michelb/Work/angular.js/docs/config/processors/pages-data.js:192:28
```